react-graphql/README.md

```diff
@@ -10,6 +10,8 @@ npm run dev
 yarn dev
 # or
 pnpm dev
+# or
+bun dev
```

Comment:
Have you used `bun` or was this added by something else? I haven't used it myself.

Reply:
That README was generated by `create-next-app`